src/Map/src/DependencyInjection/UXMapExtension.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\DependencyInjection;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\UX\Map\Bridge as MapBridge;
+
+/**
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+class UXMapExtension extends Extension implements ConfigurationInterface, PrependExtensionInterface
+{
+    /**
+     * @var array<string, array{ package: class-string<MapBridge\AbstractPackage> }>
+     */
+    private static array $bridges = [
+        'google' => [
+            'package' => MapBridge\Google\Package::class,
+        ],
+        'leaflet' => [
+            'package' => MapBridge\Leaflet\Package::class,
+        ],
+    ];
+
+    public function load(array $configs, ContainerBuilder $container): void
+    {
+        $config = $this->processConfiguration($this, $configs);
+
+        $loader = (new PhpFileLoader($container, new FileLocator(\dirname(__DIR__).'/../config')));
+        $loader->load('services.php');
+
+        $renderers = ['main' => $config['renderer']];
+        $container->getDefinition('ux_map.renderers')->setArgument(0, $renderers);
+
+        foreach (self::$bridges as $name => ['package' => $package]) {
+            if (ContainerBuilder::willBeAvailable('symfony/ux-map-'.$name, $package, ['symfony/ux-map'])) {
+                $factoryDefinition = (new ChildDefinition('ux_map.renderer_factory.abstract'))
+                    ->setClass($package::getRendererFactoryClass())
+                    ->addTag('ux_map.renderer_factory')
+                ;
+
+                $container->setDefinition('ux_map.renderer_factory.'.$name, $factoryDefinition);
+            }
```

Comment:
```suggestion
            }
```

Should we remove / disable the component / throw if no bridge is configured ?

Reply:
Let's throw if no bridge are configured

Reply:
Seems nice
<img width="1397" alt="image" src="https://github.com/user-attachments/assets/686dc06a-87c7-4186-9315-be5ab4fb1abe">

Reply:
Changed for a fallback to `NullFactory` which throws when a map is trying to be rendered, so it won't break during Symfony cache clear, and will allows the recipes to run smoothly.